rasa/model_testing.py

```diff
@@ -313,7 +314,10 @@ def perform_nlu_cross_validation(
 
     additional_arguments = additional_arguments or {}
     folds = int(additional_arguments.get("folds", 3))
-    nlu_config = rasa.nlu.config.load(config)
+
+    nlu_config = rasa.shared.importers.autoconfig.get_configuration(
```

Comment:
I assume somewhere up the chain we use a `RasaFileImporter`. Would it possible to get the config from there, as I think that calls this method at some point?

Reply:
I changed to using TrainingDataImporter but had to change the interface of `perform_nlu_cross_validation`